### PR TITLE
chore: proof docs

### PR DESCRIPTION
**Pull request by GitInVisualStudio**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -47,98 +50,98 @@ log: info
...
-      poolid: 105
-      slug: linea
+  - chainid: kaon-1
+    indexer: EthBlobs
+    poolid: 21
+    slug: ethereum
+  - chainid: korellia-2
+    indexer: Height
+    poolid: 105
+    slug: linea
```
Comment (christopherbrumm) [inline .github/README.md:60]: Please add `proofAttached`

File: .github/README.md [hunk 2]
```diff
@@ -286,13 +296,68 @@ type SaveDataItem interface {
...
 2. Now the server looksup the data item location for that key. Following our example, the server would call the database adapter with the following arguments: `Get(1337, EthBlobIndexHeight)`
-    - `EthBlobIndexHeight = 0` because the block_height is the first index defined in `EthBlobs`
+   - `EthBlobIndexHeight = 0` because the block_height is the first index defined in `EthBlobs`
 3. Now that we have the data item's location, serves the data item directly.
 4. At this point the server has provided the user with all the necessariy information to query for the on-chain merkle root for that specific data item.
 5. Finally, the user constructs the local merkle root hash based on the provided data item from the server and compares it to the on-chain merkle root.
+
+### Response Structure
+
+The Trustless API response contains the actual data item and the inclusion proof that contains all necessary information to verify the data item on-chain. The actual data of the data item might be wrapped in a custom structure, depending on the indexer. For example, the `Tendermint` indexer wraps the data item in a `JSON-RPC 2.0` response.
```
Comment (christopherbrumm) [inline .github/README.md:313]: Please change to _[...] information to verify the data item has been validated through KYVE_

File: .github/README.md [hunk 3]
```diff
@@ -286,13 +296,68 @@ type SaveDataItem interface {
...
+| poolId | 2 bytes | Pool ID (uint16) |
+| bundleId | 8 bytes | Bundle ID (uint64) |
+| chainId | variable | Chain ID (null-terminated string) |
+| dataItemKey | variable | Data Item Key (null-terminated string) |
+| dataItemValueKey | variable | Data Item Value Key (null-terminated string) |
+| Merkle Nodes | 33 bytes each | Array of Merkle nodes: <br> - 1 byte: left (true/false) <br> - 32 bytes: hash (sha256) |
+
+Note: The proof is encoded in big-endian.
+
+To construct the original data item from the trustless api response, the `dataItemKey` and `dataItemValueKey` are necessary. This is because the data item might be wrapped in a custom structure by the indexer, such as the Tendermint indexer.
```
Comment (christopherbrumm) [inline .github/README.md:342]: Please change to _Trustless API_

File: .github/README.md [hunk 4]
```diff
@@ -286,13 +296,68 @@ type SaveDataItem interface {
...
+
+Constructing the original data item from the trustless api response:
```
Comment (christopherbrumm) [inline .github/README.md:344]: Please change to _Trustless API_

File: .github/README.md (hunk 1 above)
Comment (GitInVisualStudio) [reply .github/README.md:60]: I feel like `proofAttached` should be true by default. Maybe we should rename this flag to `excludeProof` and make it optional.

**Review by christopherbrumm — APPROVED** [review]

**Pull request merged**